platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,480 @@
+
+###### Characteristics:
+
+* Technical solutions are built in tools specific to their problem domain, not always tools familiar to the users
+* There is investment in a common path, however deviating from that path quickly uncovers few customization options as the focus was on building a single option
+* Given standardization, informal internal groups are able to form and gather to share good practices and overcome shared problems
+* There may be drift on capability implementation as teams take templates, customize them and then can not merge in changes from the centralized team
+
+###### Examples Scenarios:
+
+* A centralized team curates a library of Terraform modules, Kubernetes controllers and CRDs for provisioning different types of infrastructure
+* A shared location includes comprehensive documents about solutions across the organization
+
+##### Level 3, Scalable - Self-service solutions
+
+Solutions are offered in a way that provides autonomy to users and requires little support by maintainers. The organization encourages and enables solutions to provide consistent interfaces which enables discoverability and portability of user experience from one capability to another. While self-service, the solutions do require team awareness and implementation. In order to improve this experience there may be a guided and simplified internal language which enables users to adopt and integrate platform capabilities more quickly. This generates a user-centric, self-serviceable, and consistent collection of capabilities.
+
+###### Characteristics:
+
+Solutions are provided as “one-click” implementations, enabling teams to benefit from a capability without needing to understand how they are provisioned
+While the solutions are easy to create, there may not be as much usability built into the day 2 and beyond management of the solution
+There continues to be a narrow path of available solutions, leaving users with unique requirements unsure how to proceed
+
+###### Examples Scenarios:
+
+An API is provided which abstracts the creation and maintenance of databases and provides users with any information they require to leverage that platform capability such as a connection string, location for secret data, and dashboard with observability data.
+
+##### Level 4, Optimized - Managed services
+
+Platform capabilities are transparently integrated into the tools and processes that teams already use to do their work. Some capabilities are provisioned automatically, such as observability or identity management for a deployed service. When users hit the edges of the provided services, there is an opportunity to move past automated solutions and customize for their needs without leaving the internal offerings because platform capabilities are considered building blocks which are built into the transparent and automatic compositions to meet the higher-level use cases while enabling deeper customization where necessary.
```

Comment:
```suggestion
Platform capabilities are transparently integrated into the tools and processes that teams already use to do their work. Some capabilities are provisioned automatically, such as observability or identity management for a deployed service. When users hit the edges of the provided services, there is an opportunity to move past automated solutions and customize for their needs without leaving the internal offerings because platform capabilities are considered building blocks that are built into the transparent and automatic compositions to meet the higher-level use cases while enabling deeper customization where necessary.
```